LeaderboardBackend/Services/Impl/AccountConfirmationService.cs

```diff
@@ -48,8 +48,7 @@ public async Task<CreateConfirmationResult> CreateConfirmationAndSendEmail(User
                 UserId = user.Id,
             };
 
-        _applicationContext.AccountConfirmations.Add(newConfirmation);
-
+        await _applicationContext.AccountConfirmations.AddAsync(newConfirmation);
```

Comment:
I'm guilty of making this mistake in my own code but plain Add() is actually preferred. See #199.